docs/resources/aws_emr_cluster.md

```diff
@@ -0,0 +1,106 @@
+---
+title: About the aws_emr_cluster Resource
+platform: aws
+---
+
+# aws\_emr\_cluster
+
+Use the `aws_emr_cluster` InSpec audit resource to test properties of a single AWS EMR Cluster.
+
+## Syntax
+
+An `aws_emr_cluster` resource block declares the tests for a single AWS EMR Cluster by cluster ID.
+```ruby
+describe aws_emr_cluser(cluster_id: 'CLUSTER_ID') do
+  it { should exist }
+end
+```
+```ruby
+describe aws_emr_cluser('CLUSTER_ID') do
+  it { should exist }
+end
+```
+#### Parameters
+
+##### cluster\_id _(required)_
+
+This resource requires a single parameter, the EMR Cluster ID.
+This can be passed either as a string or as a `cluster_id: 'value'` key-value entry in a hash.
+
+See also the [AWS documentation on EMR Clusters](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-what-is-emr.html).
+
+## Properties
+
+|Property                                | Description|
```

Comment:
Can you please add the other attributes available?

Link: 
1. https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/EMR/Client.html#describe_cluster-instance_method
2. https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/EMR/Client.html#describe_cluster-instance_method